accumulo/src/main/scala/geotrellis/spark/io/accumulo/AccumuloLayerHeader.scala

```diff
@@ -30,34 +32,29 @@ case class AccumuloLayerHeader(
 }
 
 object AccumuloLayerHeader {
-  implicit object AccumuloLayerMetadataFormat extends RootJsonFormat[AccumuloLayerHeader] {
-    def write(md: AccumuloLayerHeader) =
-      JsObject(
-        "format" -> JsString(md.format),
-        "keyClass" -> JsString(md.keyClass),
-        "valueClass" -> JsString(md.valueClass),
-        "tileTable" -> JsString(md.tileTable),
-        "layerType" -> md.layerType.toJson
+  implicit val accumuloLayerHeaderEncoder: Encoder[AccumuloLayerHeader] =
+    Encoder.encodeJson.contramap[AccumuloLayerHeader] { obj =>
+      Json.obj(
+        "keyClass" -> obj.keyClass.asJson,
+        "valueClass" -> obj.valueClass.asJson,
+        "tileTable" -> obj.tileTable.asJson,
+        "layerType" -> obj.layerType.asJson,
+        "format" -> obj.format.asJson
```

Comment:
`Encoder.forProductN` methods are a bit cleaner, to my eye, but this is a matter of taste and I think this looks pretty good